Fixes problem with Android Studio's Import sample feature

JoseAlcerreca opened the pull request:
Addressing #303

yigit commented:
lgtm.
just add a comment on versions.gradle to mention that script should be run
and add comment on the copied file to say do not edit this file directly, and even reference to this ticket so that someone can discover easily what was going on.

JoseAlcerreca commented:
Done, but I removed the comment from the copied file in case a developer downloads a single sample from Android Studio.

Also, fixed gradle versions so that circle only uses one process.
